### DISCUSSION TITLE
How to start learning after orientation???

### DISCUSSION THREAD

**Discussion by Sumitkcr** [github/india#249] [General]
while in the SELF-learning tab of futurereadytalent, i am unable to open any courses of AZURE.. How to access it????

**Comment by rachana-kandi** [comment]
same here

**Comment by Vaishnavi2631** [comment]
when i click on proceed course the page is not redirecting

**Reply by RomaJadhav** [reply]
I think the self learning portal is going to start tomorrow the timetable was shown in one of the slides

**Reply by Vaishnavi2631** [reply]
Ok thank you!

**Reply by SagnikPahari** [reply]
close the tab. open it in incognito mode again. it is available right now
